styles.css

```diff
@@ -0,0 +1,197 @@
+nav {
+  position: relative;
+  z-index: 99 !important;
+  background-color: #fff;
+}
+
+.nav1 {
+  width: 97%;
+  margin: auto;
+  background-color: #f72210;
+  color: #fff;
+}
+
+.nav1 p {
+  margin: auto 30px;
+}
+
+.title {
+  color: #fff;
+  font-size: 90px;
+  font-weight: bold;
+}
+
+.title:hover {
+  text-decoration: none;
+  color: #fff;
+}
+
+.nav2,
+.nav3 {
+  width: 97%;
+  margin: 10px auto;
+}
+
+.nav2 a {
+  text-decoration: none;
+  border-right: 1px solid #aaa;
+  font-weight: bold;
+  padding: 5px 15px;
+  color: #000;
+  font-size: 18px;
+}
+
+.nav3 a {
+  text-decoration: none;
+  color: #aaa;
+  font-weight: bold;
+  padding: 5px 10px;
+}
+
+.footer2 a {
+  color: #aaa;
+  text-decoration: none;
+  border-right: 1px solid lightgray;
+  padding: 0 5px;
+}
+
+.footer3 a {
+  color: #aaa;
+  text-decoration: none;
+  border-right: 1px solid lightgray;
+  padding: 0 5px;
+}
+
+.nav2 a:hover {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.nav3 a:hover {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.login {
+  background-color: #f72210;
+  border: none;
+  color: #fff;
+}
+
+.account {
+  display: none;
+}
+
+.sticky {
+  position: sticky;
+  top: 0;
+  margin-top: -140px;
+}
+
+.sticky-a {
+  padding-left: 5px;
+  color: #fff;
+  font-weight: bold;
+  font-size: 25px;
+  margin: auto 0;
+}
+
+.sticky-a:hover {
+  text-decoration: none;
+  color: #fff;
+  z-index: 1;
+}
+
+.buttons {
+  margin: auto 30px;
+}
+
+.subscribe {
+  background-color: #ffe400;
+  border: none;
+  border-radius: 25px;
+  font-weight: bold;
+  padding: 7px;
+  margin: 5px;
+}
+
+footer {
+  width: 97%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 40px;
+}
+
+.footer1 span {
+  color: gray;
+  font-size: 16px;
+}
+
+.footer-newsweek {
+  color: #f72210;
+  font-size: 40px;
+  font-weight: bolder;
+}
+
+.footer-newsweek:hover {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.icons {
+  margin: auto;
+}
+
+.icons i {
+  color: #555;
+  margin: 0 4px;
+}
+
+.icons i:hover {
+  color: #007bff;
+}
+
+.footer2 a:hover,
+.footer3 a:hover {
+  color: #000;
+}
+
+@media only screen and (max-width: 768px) {
+  .b-nav {
+    display: none;
+  }
+
+  .account {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 320px) {
+  .b-nav {
+    display: none;
+  }
+
+  .login {
+    display: none;
+  }
+
+  .account {
+    display: block;
+    margin: auto;
+  }
+
+  .bars {
+    display: block;
+    margin: auto;
+  }
+
+  .subscribe {
+    padding: 0;
+    margin: 0 3px;
```

Comment:
- [ ] Use bootstrap for this, [learn more](https://getbootstrap.com/docs/4.0/utilities/spacing/)